NickvisionCavalier.GNOME/Controls/MessageDialog.cs

```diff
@@ -51,7 +39,6 @@ public MessageDialog(Gtk.Window parentWindow, string iconName, string title, str
             _dialog.AddResponse("suggested", suggestedText);
             _dialog.SetResponseAppearance("suggested", Adw.ResponseAppearance.Suggested);
         }
-        _dialog.OnResponse += (sender, e) => SetResponse(e.Response);
```

Comment:
You could've kept this part and the `SetResponse` message. And then in PreferencesDialog in the dialog.OnResponse you could've used `dialog.Response` and compared enum instead of string with `e.Response`, but that's up to u...

Reply:
In this case we would check string anyway to select enum value, so what's the point?

Reply:
eh enum value is more restrictive...since in `MessageDialog` is an API we control, we know the expected string values and can check...someone using `MessageDialog` API that we made, may not know that, so the enum would provide a clear set of values to use. Just good design IMO then comparing strings. I try to avoid it when necessary, hence why I made the `MessageDialog` like that in the first place.